### DISCUSSION TITLE
2005/11/13/%E4%B9%85%E7%97%85%E6%88%90%E5%8C%BB

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#56] [Announcements]
# 2005/11/13/%E4%B9%85%E7%97%85%E6%88%90%E5%8C%BB

这几年生了各种各样的一些 小毛病。现将生病和治疗的精力记录下来，与病友参考。

https://blog.qizhen.xyz/2005/11/13/%E4%B9%85%E7%97%85%E6%88%90%E5%8C%BB

**Comment by ruanqizhen** [comment]
之前博客下的留言：

nicolle
2005年11月13日 18:36 编辑
例行回访
